Documentation/guides/BuildProcess.md

```diff
@@ -436,14 +436,38 @@ when packaging Release applications.
     assembly-qualified type name of an `HttpMessageHandler` subclass, suitable
     for use with
     [`System.Type.GetType(string)`](https://docs.microsoft.com/dotnet/api/system.type.gettype?view=netcore-2.0#System_Type_GetType_System_String_).
+    The most common values for this property are:
 
-    The default value is `System.Net.Http.HttpClientHandler, System.Net.Http`.
+    -   `Xamarin.Android.Net.AndroidClientHandler`: Use the Android Java APIs
+        to perform network requests. This allows accessing TLS 1.2 URLs when
+        the underlying Android version supports TLS 1.2. Only Android 5.0 and
+        later reliably provide TLS 1.2 support through Java.
 
-    This may be overridden to instead contain
-    `Xamarin.Android.Net.AndroidClientHandler`, which uses the Android
-    Java APIs to perform network requests. This allows accessing TLS 1.2 URLs
-    when the underlying Android version supports TLS 1.2.  
-    Only Android 5.0 and later reliably provide TLS 1.2 support through Java.
+        This corresponds to the **Android** option in the Visual Studio
```

Comment:
I decided to try including the UI names of each of the property values as they appear in the Visual Studio property pages. This does introduce an extra dependency for this document on the latest behavior of the Visual Studio extensions, and those extensions aren't part of the xamarin-android repository, so I could see an argument against including the UI names here. But hopefully this won't be much of a maintenance burden because the names probably won't change often.

In any case, I'd be fine with taking these UI names back out if it seems better.